content/en/getting-started.md

```diff
@@ -64,6 +64,10 @@ and you will need to repeat it several times per year as your certificate
 expires. For most people it is better to request Let's Encrypt support from your
 hosting provider, or switch providers if they do not plan to implement it.
 
+# Large Integrations
+
+If you have a large integration of Let's Encrypt certificates, please read our [Integration Guide] (https://letsencrypt.org/docs/integration-guide/).
```

Comment:
```suggestion
If you have a large integration of Let's Encrypt certificates, please read our [Integration Guide](/docs/integration-guide).
```
Same as #1013: Links must be relative (for Netlify preview and for translations) and no space between `](` 🙂

Reply:
Looks good to me, @jaykaypea